### ISSUE TITLE
updateCustomer Mutation issue with billing update

### ISSUE THREAD

**Issue by fatheaddrummer** [wp-graphql/wp-graphql-woocommerce#579]
**Describe the bug**
When the updateCustomer Mutation is called to update the billing information, the billing information is updated correctly, but the shipping information is not touched, even when you enter `shippingSameAsBilling: true`. The shipping information, however, gets updated when you call the exact same mutation a second time.

**To Reproduce**

1. Enter the following mutation and attach a `woocommerce-session` header.

```
mutation MyMutation {
  updateCustomer( input: {
      billing: {
        country: AT
      },
    shippingSameAsBilling: true
  }){
    customer{
      billing {
        country
      }
      shipping{
        country
      }
    }
  }
}

```

It should return equal billing and shipping response. But it returns this:

```
      "customer": {
        "billing": {
          "country": "AT",
        },
        "shipping": {
          "country": "DE",
        }
     }
```

2. Post the exact same mutation again and you will receive this 

```
      "customer": {
        "billing": {
          "country": "AT",
        },
        "shipping": {
          "country": "AT",
        }
      }
```

3. Change the Billing Country in the Mutation

```
mutation MyMutation {
  updateCustomer( input: {
      billing: {
        country: DE
      },
    shippingSameAsBilling: true
  }){
    customer{
      billing {
        country
      }
      shipping{
        country
      }
    }
  }
}

```

and you will receive updated billing with old shipping

```
      "customer": {
        "billing": {
          "country": "DE",
        },
        "shipping": {
          "country": "AT",
        }
      }
```

The issue can be reproduced infinitely.

**Expected behavior**
The expected behavior is to keep the shipping in sync with the billing, when billing information is updated and when `shippingSameAsBilling` is set to `true`

**Plugin Versions**
- **WooGraphQL Version:** …
- 0.10.5
- **WPGraphQL Version:** …
- 1.6.7
- **WordPress Version:**
- 5.8.1
- **WooCommerce Version:**
- 5.8.0

**Comment by kidunot89** [comment]
@fatheaddrummer That's because that field is honestly only used in `checkout` mutation and widely ignored in the CRUD order mutations. 🤔

**Comment by Amyx000** [comment]
@kidunot89 the shipping is not getting updated even in checkout mutation only the billing is updated.
```
mutation checkout {
  checkout(
    input: {paymentMethod: "bacs", 
billing: {address1: "testing123", address2: "testing12345", city: "mumbai", state: "Maharashtra", overwrite: false, country: IN},
shipping: {address1: "testingabc", address2: "tetsing", city: "mumbai", state: "MH", country: IN}}
  ) {
    order {
      shipping {
        address1
        address2
        city
      }
      billing {
        address1
        address2
        city
      }
    }
  }
}
```
```
 "shipping": {
  "address1": null,
   "address2": null,
   "city": null
   },
     "billing": {
          "address1": "testing123",
          "address2": "testing12345",
          "city": "mumbai"
        }
```

**Comment by kidunot89** [comment]
@Amyx000 @fatheaddrummer This was patched in WooGraphQL v0.15.0 by this PR #752
